Source/gg2/Objects/InGameElements/LooseSheet.events/Collision with BurningProjectile.xml

```diff
@@ -13,13 +13,18 @@
       <not>false</not>
       <appliesTo>.self</appliesTo>
       <arguments>
-        <argument kind="STRING">if(not burning) {
+        <argument kind="STRING">if(!burning
+    and sprite_index != SheetFallingGold1
```

Comment:
This is on the threshold, but I think it would be better to have a property isGold or similar to trigger these behaviour changes instead of comparing with the sprites every time.